src/ngraph/op/util/reshape.hpp

```diff
@@ -16,6 +16,9 @@
 
```

Comment:
Not in scope for this PR, but while I'm thinking about it: this stuff should be moved to `builder`

Reply:
Will do that in following PR.